lisa/tools/ping.py

```diff
@@ -46,7 +46,18 @@ def ping_async(
     ) -> Process:
         if not target:
             target = INTERNET_PING_ADDRESS
-        args: str = f"{target} -c {count} -i {interval} -O"
+        args: str = ""
```

Comment:
refer to Windows in echo, 1) update create to supports FreeBSD like `if "FreeBSD" in node.os.name` in executable.py. 3) create a new class for FreeBSD.

Reply:
update, work like a charm